### PR TITLE
add useless additions

### PR DESCRIPTION
**Pull request by ktorii**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,4 +16,6 @@ Please do not clone this repository directly.
 1. Learn by doing. 
 2. Learn how to make changes and commit to main repo.
 3. Learn how to setup ssh key.
-4. Setup 2-factor authentication on github account.
\ No newline at end of file
+4. Setup 2-factor authentication on github account.
+
```
Comment (ntorii) [inline README.md:20]: The hecka re you doing here @ktorii

File: test [hunk 2]
```diff
@@ -0,0 +1 @@
+line1edit
```
Comment (ntorii) [inline test:1]: Can you put the actual friken new line here please. Dunno why we need to. Some C problem apparently

**Review by ntorii — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (ktorii) [reply README.md:20]: nothing

File: test (hunk 2 above)
Comment (ktorii) [reply test:1]: no

**Pull request merged**